rotkehlchen/db/dbhandler.py

```diff
@@ -2032,6 +2052,10 @@ def delete_data_for_evm_address(
             'DELETE FROM evm_accounts_details WHERE account=? AND chain_id=?',
             (address, blockchain.to_chain_id().serialize_for_db()),
         )
+        write_cursor.execute(
+            'DELETE FROM key_value_cache WHERE name LIKE "extrainternaltx_%" AND value = ?',
```

Comment:
make a constant final variable

```suggestion
            'DELETE FROM key_value_cache WHERE name LIKE "{PREFIX}_%" AND value = ?',
```